projects/mc_interface/inc/wavesculptor.h

```diff
@@ -38,29 +39,266 @@ typedef struct WaveSculptorResetCmd {
+typedef struct WaveSculptorBackEmfMeasurement {
+  // Units: (V)olts
+  // Peak of the phase to neutral motor voltage.
+  float back_emf_imaginary;
+
+  // Units: (V)olts
+  // By definition this value is always 0 V.
+  float back_emf_real;
+} WaveSculptorBackEmfMeasurement;
+static_assert(sizeof(WaveSculptorBackEmfMeasurement) == 8,
+              "WaveSculptorBackEmfMeasurement is not 8 bytes");
+
+// Motor Controller Base Addr + 8
+typedef struct WaveSculptorPowerRailMeasurement {
+  // Units: (V)olts
+  // Actual voltage level of the 1.65 V analog reference.
+  float power_rail_analog_ref;
+
+  // Units: (V)olts
+  // Actual voltage level of the 15 V power rail.
+  float power_rail;
+} WaveSculptorPowerRailMeasurement;
+static_assert(sizeof(WaveSculptorPowerRailMeasurement) == 8,
+              "WaveSculptorPowerRailMeasurement is not 8 bytes");
+
+// Motor Controller Base Addr + 9
+typedef struct WaveSculptorControllerRailMeasurement {
+  // Units: (V)olts
+  // Actual voltage level of the 1.2V DSP power rail.
+  float power_rail_dsp;
+
+  // Units: (V)olts
+  // Actual voltage level of the 2.5V FPGA power rail.
+  float power_rail_fgpa;
+} WaveSculptorControllerRailMeasurement;
+static_assert(sizeof(WaveSculptorControllerRailMeasurement) == 8,
+              "WaveSculptorControllerRailMeasurement is not 8 bytes");
+
+// Motor Controller Base Addr + 10
+typedef struct WaveSculptorFanSpeedMeasurement {
+  // Units: %
+  // Drive voltage percentage to cooling fan. If this value
+  // is above 0%, then the fan should be spinning
+  float voltage_percentage;
+
+  // Units: rpm
+  // Cooling fan speed in revolutions per minute.
+  float fan_rpm;
+} WaveSculptorFanSpeedMeasurement;
+static_assert(sizeof(WaveSculptorFanSpeedMeasurement) == 8,
+              "WaveSculptorFanSpeedMeasurement is not 8 bytes");
+
+// Motor Controller Base Addr + 11
+typedef struct WaveSculptorSinkMotorTempMeasurement {
+  // Units: (C)elsius
+  // Internal temperature of the motor.
+  float motor_temp;
+
+  // Units: (C)elsius
+  // Surface temperature of the controller heatsink.
+  float heatsink_temp;
+} WaveSculptorSinkMotorTempMeasurement;
+static_assert(sizeof(WaveSculptorSinkMotorTempMeasurement) == 8,
+              "WaveSculptorSinkMotorTempMeasurement is not 8 bytes");
+
+// Motor Controller Base Addr + 12
+typedef struct WaveSculptorAirInCpuTempMeasurement {
+  // Units: (C)elsius
+  // Temperature of the internal processor.
+  float processor_temp;
+
+  // Units: (C)elsius
+  // Ambient temperature at the ventilation inlet of the controller.
+  float air_inlet_temp;
+} WaveSculptorAirInCpuTempMeasurement;
+static_assert(sizeof(WaveSculptorAirInCpuTempMeasurement) == 8,
+              "WaveSculptorAirInCpuTempMeasurement is not 8 bytes");
+
+// Motor Controller Base Addr + 13
+typedef struct WaveSculptorAirOutCapTempMeasurement {
+  // Units: (C)elsius
+  // Ambient temperature of the internal bus capacitors. Unused by 20kW WaveSculptor.
```

Comment:
If these are unused on our motor then maybe make it more explicit by adding that to the field or message name. Just to avoid accidents.